mindmap/static/js/src/mindmap.js

```diff
@@ -64,13 +59,13 @@ function loadJSMind(callback) {
         callback();
     } else {
         // Load jsMind dynamically using $.getScript
-        $.getScript("https://unpkg.com/jsmind@latest/es6/jsmind.js")
+        $.getScript("https://unpkg.com/jsmind@0.6.4/es6/jsmind.js")
```

Comment:
Why are you changing `latest`? if the library is updated we won't have the latest version

Reply:
Because the `latest` release can be a breaking change a break all the platforms with that version. Also, it's a good practice to pin the version of our dependencies

Reply:
Sure, that can be included here https://github.com/eduNEXT/xblock-mindmap/pull/12